src/google/cloud/ndb/query.py

```diff
@@ -51,38 +53,56 @@
 _GT_OP = ">"
 _OPS = frozenset([_EQ_OP, _NE_OP, _LT_OP, "<=", _GT_OP, ">=", _IN_OP])
 
+_log = logging.getLogger(__name__)
+
 
 class QueryOptions:
     __slots__ = (
-        "client",
+        # Query options
```

Comment:
I've tried to organize these according to which ones are passed to the ``Query`` constructor and which ones are passed to ``Query.fetch``. ``projection`` is passed to both.